iob.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+function calcTotal(treatments, profile, time) {
+var iob = 0
+, activity = 0;
+if (!treatments) return {};
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected '{' and instead saw 'return'.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/762667217/issues/source?bid=2452374&fileBranchId=2452471#l5)